src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -1250,15 +1250,19 @@ internal class OnboardingSupervisor(
 
     @Throws(Exception::class)
     fun subaccountTransferPayload(subaccountNumber: Int?): HumanReadableSubaccountTransferPayload {
-        val transfer = stateMachine.state?.input?.transfer ?: throw Exception("Transfer is null")
-        val size = transfer.size?.size ?: throw Exception("size is null")
-        val destinationAddress = transfer.address ?: throw Exception("destination address is null")
+        val transfer = stateMachine.state?.input?.transfer ?: error("Transfer is null")
+        val size = transfer.size?.size ?: error("size is null")
+        val destinationAddress = transfer.address ?: error("destination address is null")
+        val accountAddress = helper.parser.asString(
```

Comment:
@mike-dydx need help making sure `accountAddress` is being accessed from state properly here.